core/src/test/java/com/google/common/truth/PrimitiveByteArraySubjectTest.java

```diff
@@ -67,12 +67,11 @@ public void isEqualTo_Fail() {
       assertThat(e)
           .hasMessageThat()
           .isEqualTo(
-              "Not true that <(byte[]) [124, 112, 1, 0, 0, 1, 0, 0, 0, 0, 0, 0, 7, 101, 120, 97, "
-                  + "109, 112, 108, 101, 3, 99, 111, 109, 0, 0, 1, 0, 0]> is equal to "
-                  + "<[124, 112, 1, 0, 0, 1, 0, 0, 0, 0, 0, 0, 7, 101, 120, 97, 109, 112, 108, "
-                  + "101, 3, 99, 111, 109, 0, 0, 1, 0, 1]>; "
-                  + "expected:<...C6503636F6D000001000[1]>"
-                  + " but was:<...C6503636F6D000001000[0]>");
+              "Not true that <...25,124,112,1,[66,77,88,]124,107,61,55> "
```

Comment:
@kluever isnt JUnit shortening content when its printing ?, as you can see in _expected_ and _but was_ part.